datarobot_batch_scoring/utils.py

```diff
@@ -39,6 +39,7 @@ def verify_objectid(value):
     OptKey('host'): t.String,
     OptKey('project_id'): t.String(regex='^[A-Fa-f0-9]{24}$'),
     OptKey('model_id'): t.String(regex='^[A-Fa-f0-9]{24}$'),
+    OptKey('deployment_id'): t.String(regex='^[A-Fa-f0-9]{24}$'),
```

Comment:
why dont we just use the builtin trafaret for objectids :)

Reply:
No clue, probably question for @datarobot/batch-scoring

Reply:
We want to avoid dependencies as much as possible; `bson` or `pymongo` are too heavy. I would prefer the way you did it above.